maro/simulator/scenarios/vm_scheduling/virtual_machine.py

```diff
@@ -19,12 +19,28 @@ class VirtualMachine:
         memory_requirement (int): The memory requested by VM. The unit is (GBs).
         lifetime (int): The lifetime of the VM, that is, deletion tick - creation tick.
     """
-    def __init__(self, id: int, cpu_cores_requirement: int, memory_requirement: int, lifetime: int):
+    def __init__(
+        self,
+        id: int,
+        cpu_cores_requirement: int,
+        memory_requirement: int,
+        lifetime: int,
+        sub_id: int,
+        deployment_id: int,
+        category: int
```

Comment:
Enum?

Reply:
Since the vm category is generated by the binary dataset and it is `int` type. 
Should we convert to Enum and put into VM class?

Reply:
Enum int?